src/main.cpp

```diff
@@ -96,7 +97,8 @@ int launchCli(int argc, char *argv[])
 #ifdef Q_OS_UNIX
 void registerDBusObject(QObject *object)
 {
-    if (!QDBusConnection::sessionBus().registerObject(QStringLiteral("/io/crow_translate/CrowTranslate/") + object->metaObject()->className(), object, QDBusConnection::ExportScriptableSlots))
+    const QString object_path = QStringLiteral("/%1/").arg(QStringLiteral(APPLICATION_ID).replace(QStringLiteral("."), QStringLiteral("/")));
```

Comment:
We use camelCase as Qt itself for consistency. Also I believe that there is an overload for `QChar` in `QString::replace` (a bit cheaper and shorter)?
```suggestion
    const QString objectPath = QStringLiteral("/%1/").arg(QStringLiteral(APPLICATION_ID).replace('.', '/'));
```